server/models/Feedback.js

```diff
@@ -1,39 +1,156 @@
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
-const User = mongoose.model('User');
+const Interview = mongoose.model('Interview');
 
-const opts = {
-  // Make Mongoose use Unix time (seconds since Jan 1, 1970)
-  timestamps: { currentTime: () => Math.floor(Date.now() / 1000) },
-};
-
-const FeedbackSchema = new Schema(
-  {
-    createdAt: Number,
-    updatedAt: Number,
-    content: {
-      type: String,
+const FeedbackSchema = new Schema({
+  createdAt: Number,
+  updatedAt: Number,
+  performanceLevel: {
+    type: Number,
+  },
+  categories: {
+    communication: {
+      type: Number,
     },
-    codingRating: {
+    codeEfficiency: {
       type: Number,
     },
-    communication: {
+    codeOrganization: {
       type: Number,
     },
-    user: {
-      type: Schema.Types.ObjectId,
-      ref: 'User',
+    speed: {
+      type: Number,
     },
-    interview: {
-      type: Schema.Types.ObjectId,
-      ref: 'Interview',
+    debugging: {
+      type: Number,
+    },
+    prolemSolving: {
+      type: Number,
     },
   },
-  opts
-);
+  strengths: {
+    type: String,
+  },
+  improvements: {
+    type: String,
+  },
+  resources: {
+    type: String,
+  },
+  other: {
+    type: String,
+  },
+  feedbackCreator: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+  },
+  feedbackReciever: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+  },
+  interview: {
+    type: Schema.Types.ObjectId,
+    ref: 'Interview',
+  },
+});
+
+const Feedback = mongoose.model('Feedback', FeedbackSchema);
+
+async function addFeedback(req) {
+  const {
+    performanceLevel,
+    categories,
+    strengths,
+    improvements,
+    resources,
+    other,
+    userIdReciever,
```

Comment:
This userIdReceiver should actually always be the other user in the interview array